### ISSUE TITLE
Jaws: Default axis units should be mm

### ISSUE THREAD

**Issue by DominicOram** [ISISComputingGroup/IBEX#2965]
As a scientist I would like the jaws to specify units. Currently the jaws support module defaults units to blank, meaning that all blocks pointing at gaps/centres have no units. When changing the default confirm that:
* OPIs that have the jaws value don't have additional unit labels

**Comment by DominicOram** [comment]
Ideally the units should be the same as the underlying motor. The `unit_setter` can be used to do this.
